src/commonMain/kotlin/com/carterharrison/ecdsa/EcKeyGenerator.kt

```diff
@@ -37,8 +37,7 @@ object EcKeyGenerator {
      */
     private val random32BytePrivateKey : BigInteger
         get() {
-            val privateKeyBytes = ByteArray(32)
-            SecureRandom().nextBytes(privateKeyBytes)
-            return (BigInteger(privateKeyBytes).abs())
+            val randomBytes = SecureRandom().nextBytesOf(32)
```

Comment:
Again, why should we use kotlin crypto for random?

Reply:
Same as BigInteger. Kotlin stdlib doesn't include a security library and doesn't have any SecureRandom implementation. I could move the implementation of SecureRandom here, but I found it out of scope. Open to revisit this